logstash-input-imap.gemspec

```diff
@@ -21,11 +21,11 @@ Gem::Specification.new do |s|
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
+  s.add_runtime_dependency "logstash-core", ">= 8.10.0" # compatible with Ruby 3.1+
```

Comment:
@jsvd I think we had to remove this limitation since we don't have a fix yet, WDYT?